clippy_lints/src/utils/hir.rs

```diff
@@ -409,16 +380,15 @@ impl<'a, 'tcx: 'a> SpanlessHash<'a, 'tcx> {
                 c.hash(&mut self.s);
                 l.hash(&mut self.s);
             }
-            ExprLoop(ref b, ref i, ref j) => {
+            ExprLoop(ref b, ref i, _) => {
                 let c: fn(_, _, _) -> _ = ExprLoop;
                 c.hash(&mut self.s);
                 self.hash_block(b);
                 if let Some(i) = *i {
                     self.hash_name(&i.node);
                 }
-                j.hash(&mut self.s);
```

Comment:
Because they don't really add any value... If a desugaring is the same as a manual write then it is the same